package.json

```diff
@@ -14,7 +14,7 @@
     "start": "tsdx watch",
     "build": "tsdx build",
     "test": "tsdx test",
-    "lint": "tsdx lint",
+    "lint": "eslint",
```

Comment:
This doesn't seem right. Why could we not use `tsdx lint`?

Reply:
Yeah I created a custom eslint, but per your later comment, I will change it back to tsdx lint